onpremise/how-rd-works/data-source-results-format.md

```diff
@@ -1,5 +1,13 @@
 # **Data Source Results Format**
 
+Valid types of columns returned in results:
+TYPE_INTEGER = 'integer'
+TYPE_FLOAT = 'float'
+TYPE_BOOLEAN = 'boolean'
+TYPE_STRING = 'string'
+TYPE_DATETIME = 'datetime'
+TYPE_DATE = 'date'
```

Comment:
I thought you wanted to have this structure once in the docu, in the on prem-help. I don't mind removing it.